### DISCUSSION TITLE
Can't build the .Android project

### DISCUSSION THREAD

**Discussion by zonetecde** [AvaloniaUI/Avalonia#12130] [Q&A] [answered]
Hello,

This morning I created a new Avalonia project with VS2022, exclusively for Android.

The project worked well, I was able to test putting a button etc and I managed to launch the project on my phone.

Now when I recreate the same project (Android only), I get two compilation errors when I try to Build the .Android project (without editing the main project) :

```
NETSDK1147 to generate this project, the following workloads must be installed: android
NU1012 The platform version is not present for one or more target frameworks, even if they have specified a platform: net7.0-android   
```

Does anyone know why this error occurred and how I can get the project working again?

Thanks

ps:
In the Avalonia.Android nuget package there's a warning icon
![image](https://github.com/AvaloniaUI/Avalonia/assets/56195432/57e4224a-77f0-4119-9ae2-c572de78e054)

**Comment by jack775544** [comment, marked as answer]
In the root directory of your project try running `dotnet workload restore`. That should add all the needed workloads to start your application.

**Reply by zonetecde** [reply]
thank you!
